src/main/java/com/compilercharisma/chameleonbusinessstudio/config/SecurityConfiguration.java

```diff
@@ -13,7 +13,7 @@ public class SecurityConfiguration {
 
     @Bean
     public SecurityWebFilterChain securityWebFilterChain(
-            ServerHttpSecurity security) throws Exception {
+            ServerHttpSecurity security) {
```

Comment:
Curious why you're omitting the `throws Exception` line. Is there another way you're handling this exception?